.eslintrc.js

```diff
@@ -4,7 +4,12 @@
 module.exports = {
     root: true,
     extends: ["plugin:@eslint-community/mysticatea/es2020"],
+    parserOptions: {
+        project: true,
+    },
     rules: {
+        semi: ["error", "never"],
+        "semi-spacing": ["error", { before: false, after: true }],
```

Comment:
There's something deeply funny to me about an `eslint-community` project using ESLint for formatting.